code/solutions/architecting-templates/application-blue-green.template

```diff
@@ -0,0 +1,275 @@
+AWSTemplateFormatVersion: "2010-09-09"
+
+Description: Sample template describing an example application deployment. (uksb-1q9p31idr)
+
+Parameters:
+  AppNameTagValue:
+    Description: 'Please specify a value; minimum length: 3, maximum: 50.'
+    Type: String
+    Default: AWS CloudFormation Workshop - example app
+    MaxLength: 50
+    MinLength: 3
+
+  Env:
+    Description: The name of the environment name to use.
+    Type: String
+    AllowedValues:
+      - dev
+      - qa
+      - prod
+    Default: dev
+
+  HostedZoneStackName:
+    Description: The name of the CloudFormation stack you created for hosted zone resource(s).
+    Type: String
+    Default: cloudformation-workshop-dev-hosted-zone
+    AllowedPattern: ^[a-zA-Z]{1}[a-zA-Z0-9-]*$
+    MaxLength: 128
+    MinLength: 1
+
+  InstanceType:
+    Description: Amazon EC2 instance type to use for your instances.
+    Type: String
+    AllowedValues:
+      - t2.micro
+      - t2.small
+      - t2.medium
+      - t3.micro
+      - t3.small
+      - t3.medium
+    Default: t2.micro
+
+  LatestAmiId:
+    Description: The ID of the region-specific Amazon Machine Image to use.
+    Type: AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>
+    Default: /aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2
+
+  NameTagValue:
```

Comment:
Is this tag a project name? Not sure from description.

Reply:
It is the `Name` tag intended for resources (that support tags).

Reply:
Changes added in a new commit, thanks!